mangaki/mangaki/models.py

```diff
@@ -164,13 +164,16 @@ class Meta:
     def get_absolute_url(self):
         return reverse('work-detail', args=[self.category.slug, str(self.id)])
 
+    def get_poster(self):
```

Comment:
Maybe, we could make this into `preferred_poster_url` and use a `@property` on it?